druid-shell/src/platform/x11/application.rs

```diff
@@ -72,6 +72,14 @@ struct State {
 
 impl Application {
     pub fn new() -> Result<Application, Error> {
+        // If we want to support OpenGL, we will need to open a connection with Xlib support (see
+        // https://xcb.freedesktop.org/opengl/ for background).  There is some sample code for this
+        // in the `rust-xcb` crate (see `connect_with_xlib_display`), although it may be missing
+        // something: according to the link below, If you want to handle events through x11rb and
+        // libxcb, you should call XSetEventQueueOwner(dpy, XCBOwnsEventQueue). Otherwise, libX11
+        // might randomly eat your events / move them to its own event queue.
+        //
+        // https://github.com/xi-editor/druid/pull/1025/files/76b923417183bd103f61e56b56a56474b7417cec#r442777892
```

Comment:
This link is broken for me.

Reply:
Thanks, fixed, and I'll merge when CI lets me. Definitely nice to be using a library that is both rusty and actively maintained to the point where the maintainer drops by and tells us what we should be doing better :smile: